libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.scss

```diff
@@ -21,47 +42,27 @@
   flex-grow: 1;
 }
 
-.sky-phone-field-search-btn-dismiss {
-  border-color: transparent;
-}
-
-.sky-phone-field-country-selected,
-.sky-phone-field-country-default {
-  @include mixins.sky-border(light, bottom);
-}
-
-.sky-phone-field-search-result {
-  .sky-phone-field-flag {
-    display: inline-block;
-    margin-right: $sky-margin-half;
-  }
-}
-
+// Default only style for the dismiss icon in input boxes
```

Comment:
should this comment be with the style in the default override?

Reply:
No. That comment is for the `vertical-align` rule below it. I couldn't easily move that one into the overrides because the the input box style. It felt wrong to scope a default overrides section to input box in the phone field stylesheet.